x/gov/codec.pb.go

```diff
@@ -0,0 +1,2417 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: x/gov/codec.proto
+
+package gov
+
+import (
+	fmt "fmt"
+	_ "github.com/gogo/protobuf/gogoproto"
+	proto "github.com/gogo/protobuf/proto"
+	github_com_gogo_protobuf_types "github.com/gogo/protobuf/types"
+	_ "github.com/golang/protobuf/ptypes/timestamp"
+	github_com_iov_one_weave "github.com/iov-one/weave"
+	_ "github.com/iov-one/weave/coin"
+	io "io"
+	math "math"
+	time "time"
+)
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+var _ = time.Kitchen
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.GoGoProtoPackageIsVersion2 // please upgrade the proto package
+
+type Vote_VoteOption int32
+
+const (
+	Vote_ABSTAIN Vote_VoteOption = 0
+	Vote_YES     Vote_VoteOption = 1
+	Vote_NO      Vote_VoteOption = 2
+)
+
+var Vote_VoteOption_name = map[int32]string{
+	0: "ABSTAIN",
+	1: "YES",
+	2: "NO",
+}
+
+var Vote_VoteOption_value = map[string]int32{
+	"ABSTAIN": 0,
+	"YES":     1,
+	"NO":      2,
+}
+
+func (x Vote_VoteOption) String() string {
+	return proto.EnumName(Vote_VoteOption_name, int32(x))
+}
+
+func (Vote_VoteOption) EnumDescriptor() ([]byte, []int) {
+	return fileDescriptor_24f6e3c5f1b82a85, []int{6, 0}
+}
+
+// Electorate defines who may vote in an election. This same group can be used in many elections
+// and is stored for re-use
+type Electorate struct {
+	// Participants defines a list of all signatures that are allowed to participate in a vote
+	Participants []Participant `protobuf:"bytes,1,rep,name=participants,proto3" json:"participants"`
```

Comment:
@alpe are you sure you are running the canonical gogoprotobuf compiler?
I thought the last structs it generated had some XXX_... fields?

Reply:
Nevermind, I checked other ones on masters and they look like this one.
It's all good, as long as we all generate the same style.